task4.py

```diff
@@ -0,0 +1,12 @@
+# Задача 4: Петя, Катя и Сережа делают из бумаги журавликов. Вместе они сделали S журавликов.
+# Сколько журавликов сделал каждый ребенок, если известно, что Петя и Сережа сделали одинаковое количество журавликов,
+# а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+
+# *Пример:*
+
+# 6 -> 1  4  1
+# 24 -> 4  16  4
+#     60 -> 10  40  10
+
+s = int(input())
+print((s//6), ((s//6)*4), (s//6))
```

Comment:
Во-первых, тут нигде не нужны скобки, во-вторых, у вас трижды выполняется одна и та же операция, а значит трижды будут задействоваться ресурсы машины для ее выполнение. Чтобы такого не происходило, используют переменные.
```suggestion
var = s // 6
print(var, var * 4, var)
```